src/docs/introduction.rst

```diff
@@ -1,12 +1,102 @@
 Introduction
 ============
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac ligula et felis aliquam posuere. Etiam ac augue vel tortor porttitor placerat vitae ac libero. Nunc a efficitur lectus. Aliquam malesuada congue arcu quis luctus. Quisque pretium dolor orci, ut ullamcorper felis tincidunt eget. Duis auctor felis ut maximus facilisis. Nam eu feugiat tortor.
+SSSD stands for *System Security Services Daemon*, its the client piece
+that allows Linux to authenticate to a variety of user directories
+such as FreeIPA, 389, Microsoft Active Directory, 389DS, and other
+directory servers.
 
-Morbi ornare, ligula et volutpat mollis, nibh ante consequat nisl, vel pulvinar est mi sit amet nibh. In consectetur sit amet nunc at elementum. Mauris cursus fringilla consequat. Nulla imperdiet posuere nunc ac bibendum. Suspendisse potenti. Fusce quis purus ornare, sodales libero id, porttitor nisi. Donec pellentesque iaculis lectus eget molestie. Phasellus eleifend ex a ipsum suscipit mollis. Aenean a aliquet elit. Pellentesque lobortis orci non enim mollis, sit amet dapibus ante rhoncus. Pellentesque quis volutpat metus, ullamcorper vulputate mi. Fusce dapibus porta efficitur. Aenean fringilla varius odio nec blandit. Nulla porttitor nisi velit, id suscipit lorem sagittis et. Ut non enim id augue varius laoreet.
+So what does that mean exactly? Did you ever wonder how the same login
+worked on all of the computers in the school lab? That’s because they
+are joined to a domain that’s part of an identity management solution.
+A centralized, redundant, and secure set of servers to manage users,
+groups, policies, and more.
 
-Maecenas fermentum purus sed lacinia vestibulum. Quisque justo orci, fermentum eu massa quis, efficitur dapibus erat. Morbi tristique sem id erat ultricies, ut eleifend orci consequat. Vestibulum urna orci, pretium ut lectus quis, elementum fermentum tellus. Praesent ligula sem, consequat et efficitur eu, laoreet quis quam. Suspendisse nec lorem diam. Integer et dolor ligula. Curabitur dapibus mi quis commodo viverra. Donec ut efficitur ipsum. Integer at bibendum leo, a volutpat dolor. Cras ut pulvinar leo, in euismod libero. Vivamus nec laoreet sem. Cras hendrerit est sit amet diam vehicula scelerisque.
+This isn’t necessary when you have a handful of machines to manage but
+when scaled up to tens, hundreds, and thousands something needs to be
+in place. You certainly can force the new guy to go to each machine to
+create a user, update their password, remove a user. However, that’s
+pretty mean and a waste of time when there is a much easier way of
+doing things.
+
+The community project FreeIPA *(Free Identity Policy and Auditing)* is
+an Identity Management solution. It’s also known as Red Hat IdM, which
```

Comment:
I think adding some generic identity management definition or example may be useful here.

Reply:
I had somebody read it who isn't technical at all and not at all in the industry, after reading it, it made sense to him what an identity management solution was, are you okay with this?

Reply:
I am OK with this :)